### ISSUE TITLE
Capture

### ISSUE THREAD

**Issue by codestutis** [codestutis/Mancala#8]
edit the makeMove function to account for captures.

**Comment by konstantinapi** [comment]
hi could you assign me to this. I would like to work on it.

**Comment by KestutisBiskis** [comment]
This doesnt work on the computer side.

**Comment by konstantinapi** [comment]
have you completed aiMove function ?

**Comment by KestutisBiskis** [comment]
No, I am waiting until all the game rules are correctly implemented.

**Comment by konstantinapi** [comment]
want mee to work on anything ?

**Comment by codestutis** [comment]
you can do fix print board

**Comment by codestutis** [comment]
also I kind of might have broke the whole thing and it doesnt even run now
